### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tmm88**

### CODE REVIEW

**Comment by dinhviethoa** [comment]
Could you explain what you're trying to achieve with this change?
https://github.com/tiagmoraismorgado/buck/tree/patch-1

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 Buck
 ====
 
-Buck is a build tool. To see what Buck can do for you,
+Buck is a building tool. To see what Buck can do for you,
```
Comment (yiding) [inline README.md]: "build tool" is the correct term here, not "building tool".

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 check out the documentation at <http://buckbuild.com/>.
 
 [![Build Status](https://travis-ci.org/facebook/buck.svg)](https://travis-ci.org/facebook/buck) [![Build status](https://ci.appveyor.com/api/projects/status/v64qh0cd2cp9uto8/branch/master?svg=true)](https://ci.appveyor.com/project/Facebook/buck/branch/master)
 
+<hr/>
```
Comment (yiding) [inline README.md]: Headings delimit sections. Explicit horizontal rules are presentation markup which isn't really necessary here.

File: README.md [hunk 3]
```diff
@@ -16,6 +18,13 @@ To build Buck, run the following:
...
+
 License
 -------
 Apache License 2.0
+
+<hr/>
+
+<img src="https://lucene.apache.org/images/mantle-asf.png" width="150"></img>
+<img src="https://www.facebook.com/images/fb_icon_325x325.png" width="75"></img>
```
Comment (yiding) [inline README.md]: These images don't add anything and aren't necessary.

We are not an apache project so adding an ASF logo doesn't even make sense.

**Pull request closed without merging**